internal/cache/matchStatus.go

```diff
@@ -611,6 +615,32 @@ func (m MatchStatusCache) fromAPI(status apiXML.SportEventStatusType) protocols.
 	}
 }
 
+func (m MatchStatusCache) makeFeedStatistics(scoreboard *feedXML.Statistics) protocols.Statistics {
+	resolveHome := func(pair *feedXML.StatisticsPair) *uint32 {
+		if pair != nil {
+			return pair.Home
+		}
+		return nil
+	}
+	resolveAway := func(pair *feedXML.StatisticsPair) *uint32 {
+		if pair != nil {
+			return pair.Away
+		}
+		return nil
+	}
```

Comment:
you should check if statistics (scoreboard in your case) is nil, it it is return nil. It can panic here